Tasks/NuGetInstaller/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "Agent.Tasks",
```

Comment:
OK. I will remove this file.

From: Bryan MacFarlane [mailto:notifications@github.com]
Sent: Wednesday, March 30, 2016 5:06 AM
To: Microsoft/vso-agent-tasks <vso-agent-tasks@noreply.github.com>
Cc: Ivy Zhu <ivyzhu@microsoft.com>
Subject: Re: [Microsoft/vso-agent-tasks] add feed auth (#1450)


In Tasks/NuGetInstaller/package.json<https://github.com/Microsoft/vso-agent-tasks/pull/1450#discussion_r57876862>:

> @@ -0,0 +1,30 @@

> +{

> +  "name": "Agent.Tasks",

No reason to add this file. I'm adding support and will add this package.json for you. Also, the name should be your task, not Agent.Tasks

—
You are receiving this because you were mentioned.
Reply to this email directly or view it on GitHub<https://github.com/Microsoft/vso-agent-tasks/pull/1450/files/06fc2c763e0c1e416375bf67e729adc7d59ef1a1#r57876862>